crates/ruff_linter/src/rules/refurb/rules/if_exp_instead_of_or_operator.rs

```diff
@@ -99,3 +88,30 @@ pub(crate) fn if_exp_instead_of_or_operator(checker: &mut Checker, if_expr: &ast
 
     checker.diagnostics.push(diagnostic);
 }
+
+/// Parenthesize an expression for use in an `or` operator (e.g., parenthesize `x` in `x or y`),
+/// if it's required to maintain the correct order of operations.
+///
+/// If the expression is already parenthesized, it will be returned as-is regardless of whether
+/// the parentheses are required.
+///
+/// See: <https://docs.python.org/3/reference/expressions.html#operator-precedence>
+fn parenthesize_test<'a>(
```

Comment:
Not specific to this PR but I think we should build out some infrastructure that given an `Expr` returns a comparable `Precedence`. This would allow updating the precedence once instead of in many places (generator, parser, here, other places?)